packs/rollable-tables/10th-level-permanent-items.json

```diff
@@ -160,7 +160,7 @@
                 58,
                 63
             ],
-            "text": "Staff of Abjuration (Greater)",
+            "text": "Staff of Protection (Greater)",
             "type": 2,
```

Comment:
this one isn't changing the types?

Reply:
That may have just got lost when I did some post extract editing